packages/object-values/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = values;
   values({}); // []
   values([1, 2, 3]); // [1, 2, 3]
   values(function(a, b) {return a + b;}); // []
-  values(String('hello')); // []
+  values(String('hello')); // throw exception
```

Comment:
This is my bad (I forgot the `new`), should be
```
values(new String('hello')); // ['h', 'e', 'l', 'l', 'o']
```
But it's ok just to delete this comment entirely. It's not very helpful.